docs/wallets/cli.mdx

```diff
@@ -281,3 +281,67 @@ using the API endpoint:
 ```
 https://api.helium.io/v1/pending_transactions/YOUR_HASH_HERE
 ```
+
+---
+
+## Transfer Hotspot from 24-words Wallet
+
+If a Hotspot is transferred to a 24-words wallet, that is ok! However, most Maker Hotspot Management
+Apps do not currently support 24-words wallets.
+
+To transfer a Hotspot from a 24-words wallet to a 12-words wallet, follow the steps below.
+
+1. Create a 12-words wallet to receive the Hotspot using the
+   [Helium Hotspot App](/wallets/app-wallet) and record the Public Key.
+
+2. Download and install the [Helium Command Line Wallet](/wallets/cli-wallet#installing-cli-wallet)
+   as outlined above. and navigate to the installation directory.
+3. Import your 24-words seed phrase to the CLI Wallet.
+
+```bash
+./helium-wallet create basic --seed bip39 --output my-24-words-wallet.key
+```
+
+| argument                          | description                                                                                                       |
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `create basic`                    | create a wallet with a single key, as opposed to a [sharded wallet](/wallets/cli-wallet#sharded-wallet)           |
+| `--seed bip39`                    | import with an existing seed phrase using the `bip39` word list                                                   |
```

Comment:
can you include an example of how a user can format their words? i.e. `--seed word1 word2 word3` etc